### PR TITLE
Filter tab axis names

### PR DESCRIPTION
**Pull request by AndreySemjonov**
Filter tab axis style changed

### CODE REVIEW

**Comment by nerdCopter** [comment]
looks good on 0.2.X, but not 0.1.0,0.2.0:
image

Also, if a wide screen:
image

**Comment by AndreySemjonov** [comment]
Than looks like need to make version checker. Will take a look

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "emuflight-configurator",
   "description": "Crossplatform configuration tool for Emuflight flight control system.",
-  "version": "0.2.24",
+  "version": "0.2.25",
```
Comment (gretel) [inline package.json:4]: please leave the version bumps out of PRs. this will be done on merges to master and/or releasing from there

**Comment by gretel** [comment]
approved cause it's not a step backward. you can just continue to push changes in the same branch. approvals can be done incrementally.

**Comment by nerdCopter** [comment]
* @AndreySemjonov quick-fix for alignment issue pushed to branch.
* i think we can live with roll/pitch/yaw showing on old versions for the time being.

wide screen example:

**Pull request merged**